utils/src/main/kotlin/storage/HttpFileStorage.kt

```diff
@@ -41,6 +41,14 @@ class HttpFileStorage(
      */
     val url: String,
 
+    /**
+     * The postfix that is appended to the combination of the url and some additional path. Some storages process
+     * authentication via parameters that are within the final URL, so certain credentials can be stored in this
+     * postfix, e.g, "?user=standard&pwd=123". Thus, the final URL could be
+     * "https://example.com/storage/path?user=standard&pwd=123".
+     */
+    val urlPostfix: String = "",
```

Comment:
If we don't call this `query` now anyway, I'd propose to call it just `postfix`, as `path` is also not called `urlPath`.